test/spec-wasm2c-prefix.c

```diff
@@ -209,18 +227,24 @@ wasm_rt_externref_t spectest_make_externref(uintptr_t x) {
   return (wasm_rt_externref_t)(x + 1);  // externref(0) is not null
 }
 
-static bool is_equal_f32(f32 x, f32 y) {
```

Comment:
Given that we never reuse f32_bits or f64_bits, maybe we should just keep these functions as-is?